control-services/images/semaphore/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ansiblesemaphore/semaphore:dynamicVars
```

Comment:
This will break our current semaphore build.  We have forked the repo and built a docker container with custom changes.  Within that build we reference a specific version of semaphore

Reply:
Reverted. How is that custom image being bundled with VATER?